packages/features/portfolio/src/containers/portfolio-holdings/use-detailed-holdings.tsx

```diff
@@ -119,6 +114,18 @@ export function useDetailedHoldings() {
           presentValue: formatCryptoWithFiat(baseCurrency, b.toUnderlying()),
           earnings:
             totalEarningsWithNOTE?.toDisplayStringWithSymbol(3, true) || '-',
+          toolTipData: totalNOTEEarnings?.isPositive()
```

Comment:
@matthew-garrett can you make sure this same tool tip also shows up on grouped holdings?